File: java/src/CurrentTime.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class CurrentTime {
	public static void main(String[] args) {

		LocalDateTime currentTime = LocalDateTime.now();
		DateTimeFormatter formatter24Hr = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
		DateTimeFormatter formatter12Hr = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm:ss a");

		String formattedDate24Hr = currentTime.format(formatter24Hr);
		String formattedDate12Hr = currentTime.format(formatter12Hr);

		System.out.println("24 Hour Format: " + formattedDate24Hr);
		System.out.println("12 Hour Format: " + formattedDate12Hr);

	}
}

File: go/current-time.go
package main

import (
	"fmt"
	"time"
)

func main() {
	current_time := time.Now()

	fmt.Println("24 Hour Format: " + current_time.Format("2006-01-02 15:04:05"))
	fmt.Println("12 Hour Format: " + current_time.Format("2006-01-02 03:04:05 pm"))
}
